site_scons/site_tools/adb-conf/__init__.py

```diff
@@ -0,0 +1,199 @@
+# __init__.py for adb-conf tool
+# modeled on https://github.com/manuelnaranjo/scons-pkg-config
```

Comment:
Very cool.  What made you decide to write your own instead of using this tool directly?

Reply:
great question - because not all of the installs have pkg-config .pcs ( valijson, TileDB, faiss ) so the easiest thing was to just mimic the way autoconf does it.

Reply:
worth a comment?